Source/DafnyCore/Dafny.atg

```diff
@@ -1099,7 +1102,8 @@ SubModuleDecl<DeclModifierData dmod, ModuleDefinition parent, out ModuleDecl sub
   ( ModuleDefinition<dmod, parent, out submodule>
   | ModuleImport<parent, out submodule>
   | ModuleExport<parent, out submodule>
-    (. if (isTopLevel) SemErr(submodule.tok, "There is no point to an export declaration at the top level");
+    (. if (isTopLevel) SemErr(ErrorID.p_superfluous_export, submodule.tok.Prev,
```

Comment:
I'm wondering if this is always the "export" identifier. I hope so?

Reply:
It is. But I do worry that edits to the token or the grammar can break these.